content/docs/capabilities/getting-users-identity.md

```diff
@@ -90,7 +108,7 @@ For example:
 
 :::note
 
-When using the [hosted authenticate service](/docs/capabilities/hosted-authenticate-service), Pomerium uses the route domain as the [**Authenticate Service URL**](/docs/reference/authenticate-service-url).
+When using the [**hosted authenticate service**](/docs/capabilities/hosted-authenticate-service), use the route domain to fetch the JWKS key.
```

Comment:
The `iss` claim defines who issued the JWT, and a validation key must be fetched from there. Please also update the below instructions (I can't mark them as they're not modified in this PR). 

```suggestion
Pomerium Assertion JWT is signed by Pomerium. The upstream application should fetch the public key that can be obtained as follows: 

1. Hostname: `iss` claim from the token.
2. Path: `/.well-known/pomerium/jwks.json`
3. Scheme: `https`. 
4. Set `Accept: application/json` header. 

The returned [JWK Key Set](https://datatracker.ietf.org/doc/html/rfc7517#section-5) contains public keys Pomerium uses.

Use `kid` from the JWT to find the specific key in the JWK Key Set.
```

Reply:
Updated.

Reply:
Related: https://github.com/pomerium/internal/issues/1861